tests/conftest.py

```diff
@@ -227,7 +227,13 @@ def conda_build_test_recipe_path(tmp_path_factory: pytest.TempPathFactory) -> Pa
     # clone conda_build_test_recipe locally
     repo = tmp_path_factory.mktemp("conda_build_test_recipe", numbered=False)
     subprocess.run(
-        ["git", "clone", "https://github.com/conda/conda_build_test_recipe", str(repo)],
+        [
+            "git",
+            "clone",
+            "https://github.com/conda/conda_build_test_recipe",
+            "--branch=1.21.11",
```

Comment:
This is an unrelated change to make the tests pass.
It should probably not be picked verbatim for the `main` branch.

Reply:
since we're about to start the January releases I'm thinking we will want to change the target branch and revert this change

Reply:
Do you mean this commit in particular or the whole PR?
I'm totally fine to drop the commit -- it was just a means to get "all green".
For the PR itself, I'd like us to have another `3.28` patch release if possible.
The thing is that we don't have any working `3.28` package on conda-forge (some are already marked `broken`) and I'd like us to get one such release before we introduce further changes from the upcoming version.
Meaning, as a maintainer of infrastructure-related packages on conda-forge, I'd like to do one of the following:
1. Put out a build for a new `3.28` patch release once tagged upstream.
2. Add this PR as a patch to the recipe for the current version to get a working downstream build.
3. Mark all `3.28` builds as `broken` and thus continue to use `3.27` for now on conda-forge.

Option 1 being the most preferred.
Option 2 is not ideal since we'd have divergent behavior up- and downstream.
Option 3 would be the last resort to get things working again.
(For conda-forge we'd rather want to increase not decrease the use of linking checks and omitting the linking checks via config would only create maintenance burden and is thus not an option I'm willing to go with.)

Reply:
working with the team to figure out a release plan

added to 24.1.x milestone only because this is something that will need to be resolved and merged before the release can proceed